blocksync/reactor.go

```diff
@@ -64,8 +64,16 @@ type Reactor struct {
 func NewReactor(state sm.State, blockExec *sm.BlockExecutor, store *store.BlockStore,
 	blockSync bool) *Reactor {
 
+	// Mismatch is okay if tendermint can get new store via RPC
 	if state.LastBlockHeight != store.Height() {
-		panic(fmt.Sprintf("state (%v) and store (%v) height mismatch", state.LastBlockHeight,
+		// TODO: use logger before SetLogger?
```

Comment:
This is no longer a fatal condition, panic? log? print?

If statesync RPC is not set, then a panic is better?  Would we need to send more config.toml variables into this NewReactor() call?